tutorials/nlp/Dialogue.ipynb

```diff
@@ -0,0 +1,5269 @@
+            "  inflating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_03_22-13_01_25_169/CrossValidation/KFold_3/trainset/social_post.csv  \n",
+            "  inflating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_03_22-13_01_25_169/CrossValidation/KFold_3/trainset/social_query.csv  \n",
+            "  inflating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_03_22-13_01_25_169/CrossValidation/KFold_3/trainset/takeaway_order.csv  \n",
+            "  inflating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_03_22-13_01_25_169/CrossValidation/KFold_3/trainset/takeaway_query.csv  \n",
+            "  inflating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_03_22-13_01_25_169/CrossValidation/KFold_3/trainset/transport_query.csv  \n",
+            "  inflating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_03_22-13_01_25_169/CrossValidation/KFold_3/trainset/transport_taxi.csv  \n",
+            "  inflating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_03_22-13_01_25_169/CrossValidation/KFold_3/trainset/transport_ticket.csv  \n",
+            "  inflating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_03_22-13_01_25_169/CrossValidation/KFold_3/trainset/transport_traffic.csv  \n",
+            "  inflating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_03_22-13_01_25_169/CrossValidation/KFold_3/trainset/weather_query.csv  \n",
+            "   creating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_03_22-13_01_25_169/CrossValidation/KFold_4/\n",
+            "   creating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_03_22-13_01_25_169/CrossValidation/KFold_4/testset/\n",
+            "   creating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_03_22-13_01_25_169/CrossValidation/KFold_4/testset/annotated/\n",
+            "   creating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_03_22-13_01_25_169/CrossValidation/KFold_4/testset/annotated/alarm_query/\n",
+            "  inflating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_03_22-13_01_25_169/CrossValidation/KFold_4/testset/annotated/alarm_query/alarm_query_anno_20180322130134494.json  \n",
+            "   creating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_03_22-13_01_25_169/CrossValidation/KFold_4/testset/annotated/alarm_remove/\n",
+            "  inflating: NLU-Evaluation-Data-master/CrossValidation/autoGeneFromRealAnno/autoGene_2018_0
```

Comment:
The pad_token here could use some explanation.

Reply:
Sure